CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 - Based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.4.0
+2024-11-29
+
+### Changed
+- Builds `kind` nodes locally for integration tests and sandbox scripts, instead of using pre-built images.
+- Upgrades Go to 1.22.9.
```

Comment:
Can't we use [1.23](https://go.dev/blog/go1.23)?

Reply:
We can - but leaving that until the upcoming changes for the `resize` subresource. Upgrading to 1.23 here was specifically to avoid warnings being thrown for some of the dependency versions we're using.